content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
-that both teams cannot access each other's services.
-
-Let us consider a three-tier application with three services: `photo-frontend`,
-`photo-backend`, and `datastore`. The photo SRE team manages the
-`photo-frontend` and `photo-backend` services while the datastore SRE team
-manages the `datastore` service. The `photo-frontend` service can access
-`photo-backend`, and the `photo-backend` service can access `datastore`.
-However, the `photo-frontend` service cannot access `datastore`.
-
-In this scenario, a cluster administrator creates two namespaces:
-`photo-ns` and `datastore-ns`. The administrator has
-access to all namespaces and each team only has access to its own namespace.
-The photo SRE team creates two service accounts to run `photo-frontend` and
-`photo-backend` respectively in the `photo-ns` namespace. The datastore SRE
-team creates one service account to run the `datastore` service in the
-`datastore-ns` namespace. Moreover, we need to enforce the service access
-control in [Istio Mixer](https://istio.io/v1.6/docs/reference/config/policy-and-telemetry/) such that
-`photo-frontend` cannot access datastore.
-
-In this setup, Kubernetes can isolate the operator privileges on managing the services.
-Istio manages certificates and keys in all namespaces
-and enforces different access control rules to the services.
+Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization and audit (AAA) tools to protect your services and data.
+However, to fully make use of these features securely, care must be taken to follow best practices. It is recommended to review the [Security overview](/docs/concepts/security/) before proceeding.
+
+## Mutual TLS
+
+Istio will [automatically](/docs/ops/configuration/traffic-management/tls-configuration/#auto-mtls) encrypt traffic using [Mutual TLS](/docs/concepts/security/#mutual-tls-authentication) whenever possible.
+However, proxies are configured in [permissive mode](/docs/concepts/security/#permissive-mode) by default, meaning they will accept both mutual TLS and plaintext traffic.
+
+While this is required for incremental adoption or allowing traffic from clients without an Istio sidecar, it also weakens the security stance.
+It is recommended to [migrate to strict mode](/docs/tasks/security/authentication/mtls-migration/) when possible, to enforce that mutual TLS is used.
+
+Mutual TLS alone is not always enough to fully secure traffic, however, as it provides only authentication, not authorization.
+This means that anyone with a valid certificate can still access a service.
+
+To fully lock down traffic, it is recommended to configure [authorization policies](/docs/tasks/security/authorization/).
+These allow creating fine-grained policies to allow or deny traffic. For example, you can allow only requests from the `app` namespace to access the `hello-world` service.
+
```

Comment:
After mutual TLS, I think it'd be good to have a section for using the beta security policies.  

We should cover the limitations of ALLOW policies / describe when it's safer to use DENY.

We should also describe how authentication policies are decoupled from authorization policies and make it clear that authentication policies without corresponding authorization policies are just security theater.

@yangminzhu @myidpt @incfly

Reply:
I would need help with "We should cover the limitations of ALLOW policies / describe when it's safer to use DENY.", if one of the people mentioned above could take this on?

I could take on the later part if needed